examples/client.go

```diff
@@ -348,7 +348,10 @@ func main() {
 			instId = k
 			break
 		}
-		resp, err := r.AddInstances(aurora.InstanceKey{job.JobKey(), instId}, numOfInstances)
+		resp, err := r.AddInstances(aurora.InstanceKey{
```

Comment:
Push the second arg onto it's own line.
i.e:
```
resp, err := r.AddInstances(aurora.InstanceKey{
			JobKey:     job.JobKey(),
 			InstanceId: instId,
 		}, 
    numOfInstances)
```

Reply:
done